### PR TITLE
lithp ith now a roundthart option

### PR DESCRIPTION
**Pull request by PrimeDSS13**
thombody wanted thith I know it

:cl:
 * rscadd: lithp ith now a roundthart option

### CODE REVIEW

**Comment by APCUnlocked** [comment]
Now THIS is content.

File: __DEFINES/setup.dm [hunk 1]
```diff
@@ -502,6 +503,7 @@ var/global/list/NOIRMATRIX = list(0.33,0.33,0.33,0,\
 #define NERVOUS			16
 #define ASTHMA		32
 #define LACTOSE		64
+#define LISP 		128
```
Comment (DamianX) [inline __DEFINES/setup.dm]: You don't seem to be using this anywhere. Why is this here?

Comment (PrimeDSS13) [reply __DEFINES/setup.dm]: I was just copying the format of asthma honestly, if it's unnecessary I'll cut it out.

Comment (boy2mantwicethefam) [reply __DEFINES/setup.dm]: From my limited understanding of code I'm pretty sure this is used for Genetics

Comment (DamianX) [reply __DEFINES/setup.dm]: delet

**Comment by Gazathor** [comment]
the crippled femstatics get more powerful every day...
